src/client/components/forms/userCollection.js

```diff
@@ -160,6 +172,24 @@ class UserCollectionForm extends React.Component {
 		return (
 			<div>
 				<h1>Create your collection</h1>
+				<DeleteCollectionModal
+					closeDeleteModalCallback={this.closeDeleteModalCallback}
+					collection={this.props.collection}
+					show={this.state.showModal}
+				/>
+				{
+					this.props.collection.id ?
+						<Button
```

Comment:
The placement of this button looks a bit random at the moment, and in some screen sizes something comes over the button and prevents me from clicking it.
I think it would make more sense to group it with the submit button at the bottom of the page, in a `<ButtonGroup>` element.

Reply:
While you're in there, you could also add a "save" icon to the submit button :)